CustomApps/lyrics-plus/Utils.js

```diff
@@ -78,7 +78,9 @@ class Utils {
 		}
 
 		// translate it to Simplified Chinese
-		return Utils.#translator.convertChinese(s, "tw", "cn");
+
```

Comment:
Better to remove this unnecessary newline

Reply:
Alright I'll do it later 👍